From 2daf0c9d21e408c9c77fbbcab84e3c76d7ba8fa2 Mon Sep 17 00:00:00 2001
From: chromium-internal-autoroll
 <chromium-internal-autoroll@skia-corp.google.com.iam.gserviceaccount.com>
Date: Mon, 25 Feb 2019 08:40:58 +0000
Subject: [PATCH] Roll src-internal 10801c9b841b..b2c29eff50a2 (1 commits)

https://chrome-internal.googlesource.com/chrome/src-internal.git/+log/10801c9b841b..b2c29eff50a2


Created with:
  gclient setdep -r src-internal@b2c29eff50a2

The AutoRoll server is located here: https://autoroll-internal.skia.org/r/src-internal-chromium-autoroll

Documentation for the AutoRoller is here:
https://skia.googlesource.com/buildbot/+/master/autoroll/README.md

If the roll is causing failures, please contact the current sheriff, who should
be CC'd on the roll, and stop the roller if necessary.


TBR=mmoss@chromium.org

Change-Id: I30591bf5d070bc13804bb3c7d443841eef18cd0b
Reviewed-on: https://chromium-review.googlesource.com/c/1484620
Reviewed-by: chromium-internal-autoroll <chromium-internal-autoroll@skia-corp.google.com.iam.gserviceaccount.com>
Commit-Queue: chromium-internal-autoroll <chromium-internal-autoroll@skia-corp.google.com.iam.gserviceaccount.com>
Cr-Commit-Position: refs/heads/master@{#635048}
---
 DEPS | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/DEPS b/DEPS
index ab587ea7d634c..a7fbd81324d9d 100644
--- a/DEPS
+++ b/DEPS
@@ -1306,7 +1306,7 @@ deps = {
     Var('chromium_git') + '/v8/v8.git' + '@' +  Var('v8_revision'),
 
   'src-internal': {
-    'url': 'https://chrome-internal.googlesource.com/chrome/src-internal.git@10801c9b841b0dde48aeef1164551b52e129d877',
+    'url': 'https://chrome-internal.googlesource.com/chrome/src-internal.git@b2c29eff50a24a9f47996ebb7ed419412dbec1cf',
     'condition': 'checkout_src_internal',
   },
 
